packages/nuxt/src/imports/module.ts

```diff
@@ -34,14 +35,16 @@ export default defineNuxtModule<Partial<ImportsOptions>>({
 
     // Create a context to share state between module internals
     const ctx = createUnimport({
-      presets,
-      imports: options.imports,
-      virtualImports: ['#imports'],
       addons: {
-        vueTemplate: options.autoImport
-      }
+        vueTemplate: options.autoImport,
+        ...options.addons
+      },
+      ...options,
```

Comment:
as we are overriding addons with `...options` at the end, it looks like this line will never be applied.

```suggestion
        ...options.addons
      },
```